File: templates/java/es/text/code/stepUp2.java
/*
 * Archivo: ConoRecogiendo Karel .java
 * ----------------------------------
 * El programa ConoRecogiendo Karel define un método
 * " run " con tres comandos. Estos comandos causan K
 * arel a moverserse avanzar un bloque, recoger un cono
 * , y luego moverserse avanzar a la siguiente esquina.
 */
import stanford.karel.*;
public class ConoRecogiendoKarel extends Karel {
   public void run() {
      moverse();
      recogerCono();
      moverse();
      girarIzquierda();
      moverse();
      girarDerecha();
      moverse();
      moverse();
      ponerCono();
      moverse();
   }

   /**
    * Gira Karel 90 grados a la derecha.
    */
   private void girarDerecha() {
      girarIzquierda();
      girarIzquierda();
      girarIzquierda();
   }
}

File: templates/python/es/text/code/stepUp2.py
# Archivo: Cono Recogiendo Karel .py
# -----------------------------
# El programa Cono Picking Karel define un " main "
# función con tres comandos. Estos comandos causan
# Karel a moverse hacia adelante un bloque, tome un
# cono y luego moverse adelante a la siguiente esquina.
from karel.stanfordkarel import *
def main():
   moverse()
   recoger_cono()
   moverse()
   girar_izquierda()
   moverse()
   girar_derecha()
   moverse()
   moverse()
   poner_cono()
   moverse()

def girar_derecha():
   girar_izquierda()
   girar_izquierda()
   girar_izquierda()